config/jobs/update-secrets/update-secrets.yaml

```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  falcosecurity/test-infra:
+    - name: update-secrets-post-submit
+      branches:
+        - ^master$
+      decorate: true
+      max_concurrency: 1
+      run_if_changed: '^config/secrets.yaml$'
+      spec:
+        containers:
+          - image: ghcr.io/falcosecurity/pigeon:v0.3.0
```

Comment:
We also need 3 env variables to be defined to connect to our 1password store: https://github.com/falcosecurity/pigeon/blob/main/pkg/pigeon/secrets_onepassword.go#L35

```
//   - `OP_CONNECT_TOKEN`: the API token to be used to authenticate the client
//     to your 1Password Connect instance.
//   - `OP_CONNECT_HOST`: the hostname of your 1Password Connect instance.
//   - `OP_VAULT`: a vault UUID from which retrieving the secrets.
```